python/cudf/cudf/core/column/column.py

```diff
@@ -2078,11 +2078,11 @@ def as_column(
                             data
                         )
                     dtype = pd.api.types.pandas_dtype(dtype)
-                    np_type = np.dtype(dtype).type
+                    np_type = cudf.dtype(dtype).type
```

Comment:
Can we now squeeze these two separate dtype calls into a single `cudf.dtype` call? or is there something specific about calling `pd.api.types.pandas_dtype` first?

```python
np_type = cudf.dtype(dtype).type
```

Reply:
Fixed